py/server/deephaven/table.py

```diff
@@ -3761,3 +3763,64 @@ def multi_join(input: Union[Table, Sequence[Table], MultiJoinInput, Sequence[Mul
             table() method.
     """
     return MultiJoinTable(input, on)
+
+
+# region utility functions
+
+def table_diff(t1: Table, t2: Table, max_diffs: int = 1, floating_comparison: Literal['exact', 'absolute', 'relative'] = 'exact',
+               ignore_column_order: bool = False) -> str:
+    """Returns the differences between this table and the provided table as a string. If the two tables are the same,
```

Comment:
At some point in the docstring, it should indicate that an empty string is returned if the tables are the same.

Reply:
It is there. Do you mean we need to highlight it somewhere else as well?
```Python
"""Returns the differences between this table and the provided table as a string. If the two tables are the same,
    an empty string is returned. The differences are returned in a human-readable format."""
```

Reply:
I"m an idiot.